crates/biome_js_analyze/tests/specs/nursery/useGuardForIn/invalid.js.snap

```diff
@@ -0,0 +1,110 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: invalid.js
+---
+# Input
+```jsx
+for (var x in o) { if (x) { f(); continue; } g(); }
+for (var x in o) { if (x) { continue; f(); } g(); }
+for (var x in o) { if (x) { f(); } g(); }
+for (var x in o) { if (x) f(); g(); }
+for (var x in o) { foo() }
+for (var x in o) foo();
+```
+
+# Diagnostics
+```
+invalid.js:1:1 lint/nursery/useGuardForIn ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Require `for-in` loops to include an `if` statement.
+  
+  > 1 │ for (var x in o) { if (x) { f(); continue; } g(); }
+      │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    2 │ for (var x in o) { if (x) { continue; f(); } g(); }
+    3 │ for (var x in o) { if (x) { f(); } g(); }
+  
+  i The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.
```

Comment:
Our linter diagnostics must follow the rule pillars: https://biomejs.dev/linter/#rule-pillars 

This is also present in our contribution guide, please get familiar with it.

In this case, we fail to explain the error, the second pillar.

Reply:
Sorry, I referred too much to eslint here.. its message is very simple. i updated the message here..